.codeclimate.yml

```diff
@@ -0,0 +1,4 @@
+exclude_patterns:
+- "dist/"
+- "**/node_modules/"
+- "*.test.tsx"
```

Comment:
For now, it's probably worth also including .js files as the docs-site isn't ts yet.